docs/ReleaseProcess.md

```diff
@@ -0,0 +1,89 @@
+
+   **Please note that:**
+
+   - The _target_ **must** be `master`.
+   - The _version number_ **must** adhere to Semantic Version specification.
+   - The _version tag_ **must** begin with a `v` in order to indicate it is a version. Thus the full tag should be of the format `v1.2.3` for version.
+   - For _release title_ please also put the identical version tag as seen above. (If left blank GitHub auto-generates very poorly.)
+
+   SBT is configured to use these version tags as the source of truth, so treat them accordingly. :smile: In the future, we'll have a script to help out with pre-populating these values.
+
+   > **How to pick a version:** In most cases, you'll be doing a "patch" increment, which means just incrementing the last digit of the version. If the current version tag is `v0.1.29`, the next patch would be `v0.1.30`. If you think the release requires a larger Semantic Version bump (API changes for example), talk to others on `#dev` in Slack to confirm.</small>
+
+3. When you're ready to publish your release, click **Publish release**. To work on the release later, click **Save draft**.
+   ![screenshot](https://help.github.com/assets/images/help/releases/release_buttons.png)
+
+<!--
+Steps for open-law client below. Preserving in comments here since hopefully we can have something similar in future.
+
+1. Switch to `master` and pull the latest.
+
+1. Run `npm run release` to start the release process.
+
+1. A prompt will appear in your console allowing you to pick what sort of Semantic Version increment to pick. In most cases, you will be selecting a `patch` release. _(Note: you can skip the interactive prompt for a patch by substituting `npm run release:patch` in the step above.)_
+
+1. You'll see the release steps going through in your console.<sup>§</sup> When that's done the `openlaw-client` repo should open in your web browser with a draft release. All you have to do there is verify the release notes and click publish.
+-->
+
+### Publishing the release to BinTray
+
+#### Automatically (coming soon!)
+
+1. Once a GitHub Release is published, our CI system will automatically start the process to publish the package to BinTray. You can [monitor the progress for that process](#TODO_URL) and see when it completes.
+
+2. Confirm that the package was successfully published with the latest version number by [checking the BinTray listing](https://bintray.com/openlawos/openlaw-core/openlaw-core).
+
+#### Manually
+
+<!-- If for some reason, the publish to BinTray must be done manually in your local environment, you can do it via the following steps in an emergency: -->
+
+In the very near future, the following steps will be automated and done in CI, but for _right now_ we're doing them manually while we test:
+
+1. Switch to `master` and pull the latest. You may wish to also run `git fetch` to ensure you have the latest tags.
```

Comment:
-> You should also run `git fetch`

Reply:
I'd be interested to test this -- I _think_ `git pull` might automatically get any new tags if they are "reachable" from the commits you are pulling, e.g. fetch would only be required in the instance where you had the latest on your local copy before the tags were added to it. (But perhaps we should just ask people to always run fetch to be on the safe side.)

Reply:
You're probably right! 
I was more thinking of asking for completeness so we don't have to manually add tags again

Reply:
I agree, better safe than sorry.  I'll make it very explicit when I do a revision pass on folks comments.